### ISSUE TITLE
Rule docstrings trigger a syntax error

### ISSUE THREAD

**Issue by wjv** [JetBrains-Research/snakecharm#31]
Snakemake explicitly supports docstrings (bare strings as the first element in a rule, analogous to docstrings in Python). See:

https://bitbucket.org/snakemake/snakemake/issues/398/making-the-code-and-docstring-of-each-rule

Currently SnakeCharm marks a docstring as a syntax error (“Rule parameter identifier is expected”).

**Comment by wjv** [comment]
Let me amend this: It _sometimes_ triggers a syntax error. I am trying to isolate when exactly this happens.

**Comment by iromeo** [comment]
Thanks 

Code example:
```
rule a:
    """ the docstring """
    input: ...
    ...
```

**Comment by iromeo** [comment]
Issue demonstration + how *.py compilation looks like
![image](https://user-images.githubusercontent.com/72933/60290738-10b8e000-9922-11e9-8623-82db1ea9e22a.png)

**Comment by iromeo** [comment]
P.S: also let's ensure that fix also resolves errors in code like
```
rule a:
    # boooo
    """ the docstring """
    input: "foo"
```

**Comment by iromeo** [comment]
Update:
Also valid snakemake syntax:
```
rule a:
    " snakemake also treats this as doc string "
    input: "foo"
```

**Comment by iromeo** [comment]
Fixed, will be available in next major update (0.4.x)
